doc/introduction.rst

```diff
@@ -1,59 +1,135 @@
+.. _introduction:
+
+===========================
 Getting started with Binder
 ===========================
 
-An interactive introduction to Binder
--------------------------------------
+What is the Binder Project?
+===========================
+
+The Binder Project is an open community that makes it possible to create sharable,
+interactive, reproducible environments. The main technical product that
+the community creates is called BinderHub, and one deployment of a BinderHub
+exists at ``mybinder.org``. This website is run by the Binder Project as
+a public service to make it easy for others to share their work.
+
+.. _what-is-a-binder:
+
+What is a Binder?
+=================
+
+A Binder (also called a Binder-ready repository) is a code repository that
+contains at least two things:
+
+1. **Code or content that you'd like people to run.** This might be a
+   Jupyter Notebook that explains an idea, or an R script that makes a
+   visualization.
+2. **Configuration files for your environment.** These files are used
+   by Binder to build the environment needed to run your code.
+   For a list of all configuration files available, see the
+   :ref:`config-files` page.
+
+Configuration files may be placed in the root of
+your repository or in a ``binder/`` folder in the repository's root
+(i.e. ``myproject/binder/``).
+
+A Binder repository can be built by a BinderHub, which will generate a
```

Comment:
Personally, I find this language a bit confusing. Even though you defined "Binder" as a repository that can be hosted on a BinderHub, my intuition uses "Binder" and "Binder Project" interchangeably here, so my first read of this sentence is that a "Binder repository" is a repository that is a part of the binder project, which is not the intention. 

Elsewhere, the language "your Binder" is used, which I would define as "my repo" + "a BinderHub". Would it work to define "a Binder" as "a Binder-ready repo" + "a BinderHub"?

For this sentence, would it be too strange to introduce the use of the word "Binderizable". So a "Binderizable repository" is a repo that can be built on BinderHub? Or alternatively using "Binder-ready" would clear up the language for me.  

Just my 2 cents, if you feel strongly about the language, I can work on adjusting my intuition :)